building-pyramids/java/printingloops2.java

```diff
@@ -0,0 +1,16 @@
+public class printingloops2{
```

Comment:
In Java, class names are generally PascalCased, so this should be PrintingLoops2